### ISSUE TITLE
Getting started command:toSide seems quite strange to me

### ISSUE THREAD

**Issue by eamodio** [microsoft/vscode#122624]
Testing #122275

I understand the intent of `command:toSide`, but it feels awkward and like it would only work with very specific commands.

**Comment by JacksonKearl** [comment]
I agree it wouldn't make sense to use it with a command that doesn't open some sort of editor input. We could alternatively use some bespoke command to implement opening an editor input to side, but I imagine it'd be hard for ext authors to even know how to write the editor input (we probably don't want to expose internal editor resource URI's).

The `toSide` modifier is implemented like:

- Check if editor is split
  - if not, and the width is bigger than some fixed value (like 600px), split it, and make the getting started group small
-  Activate and focus a non-getting started editor group
- Run the command

Do you have any ideas for making it less awkward?

cc @digitarald

**Comment by digitarald** [comment]
`:toSize` has a sweet spot for commands that open a new tab right away.

As this issue also appears when user open an extension page from search and the many other ways that a walkthrough can eventually lead to more tabs opening – alternative Getting Started could monitor when it becomes hidden and counteract by moving itself into foreground/split view when needed. Maybe there is a less aggressive middle ground though.
